tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/retry/ConfigPropertyGlobalVsClassVsMethodTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
```

Comment:
You should not change the starting year of the copyright but just the ending year. It should be 2017-2020.